zebra/kernel_netlink.h

```diff
@@ -23,6 +23,7 @@
 
 #ifdef HAVE_NETLINK
 
+#define NL_RCV_PKT_BUF_SIZE     32768
 #define NL_PKT_BUF_SIZE         8192
 
```

Comment:
why having two values  ?
it seems it is not rcv and tx since I see NL_PKT_BUF_SIZE used in other places..

Reply:
We are not transmitting anything to the kernel of greater than 8k size already, there is no point in making this bigger at this point in time.  Can you show me where we are?  If so we can increase that buffer size.

the NL_RCV_PKT_BUF_SIZE is set to a somewhat absurdly high number because we kept on receiving data where before the 8k size was causing data to be missed from the kernel.

Reply:
ok. I misunderstood the meaning of those two values.
I was thinking one was for tx and one other for rx.
actually, this is not used in that way.
thanks for clarificaiton.